staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -1,6 +1,27 @@
 # NOTE: Please also update staging/cse/windows/provisioningscripts/loggenerator.ps1 when collecting new logs.
 $ProgressPreference = "SilentlyContinue"
 
+function CollectLogsFromDirectory {
+  Param(
+      [Parameter(Mandatory=$true)]
+      [String] $path,
+      [Parameter(Mandatory=$true)]
+      [String] $targetFileName
+  )
+  try {
+    if (Test-Path $path) {
+      Write-Host "Collecting logs from $path"
+      Compress-Archive -LiteralPath $path -DestinationPath "$ENV:TEMP\\$targetFileName"
```

Comment:
Why don't we just copy the folder? We need to unzip them recursively after we collected the log.

Reply:
There may be multiple folders and files in the path and the sizes of the dump files may be big.